src/Apps/Artist/Routes/Overview/__tests__/index.test.tsx

```diff
@@ -209,6 +209,15 @@ describe("OverviewRoute", () => {
     })
   })
 
+  describe("ConsignButton", () => {
+    it("shows a consign button", () => {
+      const wrapper = getWrapper({
+        ...defaultArtist,
+      })
+      expect(wrapper.find("ArtistConsignButton").length).toEqual(1)
```

Comment:
This assertion is pretty tied to our specific implementation. I know we are using enzyme and not testing-library, but I like to use the [testing-library guiding principles](https://testing-library.com/docs/guiding-principles) because I think their approach results in more robust tests & easier refactoring. In this case, it would mean asserting that specific text or a button with specific text appeared on the page, instead of a specific component type.

No changes expected here - It's opinionated feedback.

Reply:
Will update 👍

Reply:
These assertions are encapsulated / tested in the button itself, up a level; since this is the container we're just checking for the existence of the button on the page.